### PR TITLE
Roomba is more expressive

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Roomba will tell you how disappointed he is that you left your trash for him to pickup

## Why It's Good For The Game

Immersion

## Changelog
:cl:
qol: Roomba is more expressive
/:cl:

### CODE REVIEW

**Comment by Oscari-pog** [comment]
Wouldnt he be happy

**Comment by BraveMole** [comment]
> Wouldnt he be happy

I can make a happy version, but i need sentences

**Comment by BraveMole** [comment]
> 2%'s kinda low

if i get way more sentences, i'll make it more frequent. Don't hesitate

File: code/game/objects/machinery/roomba.dm [hunk 1]
```diff
@@ -79,13 +89,17 @@
 ///Called when the roomba moves, sucks in all items held in the tile and sends them to cryo
 /obj/machinery/roomba/proc/suck_items()
 	SIGNAL_HANDLER
+	var/sucked_one = FALSE
```
Comment (mrhugo13) [inline code/game/objects/machinery/roomba.dm:92]: the best var this codebase has so far

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**